graphics.h

```diff
@@ -38,8 +38,7 @@ class Texture {
 	const BitmapData &m_bitmap;
 };
 
-using Point = std::pair<GLdouble, GLdouble>;
-enum Coord {
-	X = 0,
-	Y = 1
-};
\ No newline at end of file
+struct Point {
+	GLdouble x;
+	GLdouble y;
```

Comment:
It turns out that structured binding is really smart and works on aggregates like this too, so we don't even need this to be a pair to get `auto [x, y] = point`.